packages/commons/src/config/featureFlagsConfig.ts

```diff
@@ -11,10 +11,15 @@ export const FeatureFlagsConfig = z
       .transform((value) => value.split(","))
       .pipe(z.array(z.string().uuid()))
       .optional(),
+    FEATURE_FLAG_REMOVE_CLIENT_ADMIN: z
```

Comment:
```suggestion
    FEATURE_FLAG_ADMIN_CLIENT: z
```
The feature flag is for the whole feature, not specifically for the endpoint